R/plot_aggregate_loanbooks.R

```diff
@@ -171,50 +171,27 @@ plot_aggregate_loanbooks <- function(config) {
         na = ""
       )
 
-    plot_sankey(
-      data_sankey_sector,
-      group_var = by_group,
-      save_png_to = path.expand(analysis_aggregated_dir),
-      png_name = glue::glue("plot_{output_file_sankey_sector}.png"),
-      nodes_order_from_data = TRUE
+    p_sankey <- plot_sankey(
+      data = data_sankey_sector,
+      y_axis = "loan_size_outstanding",
+      initial_node = by_group,
+      middle_node = "sector",
+      end_node = "is_aligned",
+      stratum = "is_aligned"
     )
-  } else {
-    cli::cli_warn("Sankey plot (by sector) cannot be generated. Skipping!")
-  }
 
-  if (!is.null(company_aggregated_alignment_net)) {
-    data_sankey_company_sector <- prep_sankey(
-      company_aggregated_alignment_net,
-      region = "global",
-      year = start_year + time_frame_select,
-      group_var = by_group,
-      middle_node = "name_abcd",
-      middle_node2 = "sector"
+    ggplot2::ggsave(
+      plot = p_sankey,
+      filename = glue::glue("plot_{output_file_sankey_sector}.png"),
```

Comment:
where does `output_file_sankey_sector` come from in this updated script?

Reply:
lines 159-163